src/main/frontend/components/FilmTile.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const FilmTile = (props) => {
+  const { id, title, imgUrl, year } = props.film
+
+  return (
+    <div>
+      <a href={`/films/${id}`}>
```

Comment:
Let's use a `Link` component instead of an anchor tag to take advantage of React!

Reply:
Replaced href with a commented-out line to satisfy Fang's comment.